dev_guide/builds.adoc

```diff
@@ -576,6 +576,28 @@ Specify the `--follow` flag to stream the build's logs in stdout:
 $ oc start-build <BuildConfigName> --follow
 ----
 
+Specify the `--env` flag to set environment variable for the build:
+
+----
+$ oc start-build <BuildConfigName> --env=RACK_ENV=development
+----
```

Comment:
Should we update other sections and mention about it? For instance:
- https://docs.openshift.org/latest/dev_guide/builds.html#configuring-the-source-environment
- https://docs.openshift.org/latest/dev_guide/builds.html#docker-strategy-environment
- https://docs.openshift.org/latest/dev_guide/builds.html#custom-strategy-environment